Formula/maxbin2.rb

```diff
@@ -0,0 +1,39 @@
+class Maxbin2 < Formula
+    # cite Wu_2016: "https://doi.org/10.1093/bioinformatics/btv638"
+    desc "an automated binning algorithm to recover genomes from multiple metagenomic datasets"
+    homepage "https://bioconda.github.io/recipes/maxbin2/README.html"
+    url "https://sourceforge.net/projects/maxbin2/files/MaxBin-2.2.5.tar.gz/download"
+    sha256 "4fb248ec5173ce93055e72d9d806f2037b4e12f10c5c643b5ba7d265bb7c1459"
+  
+    depends_on "cmake" => :build
+    depends_on "cpanminus" => :build
+    depends_on "bowtie2"
+    depends_on "hmmer"
+    depends_on "idba"
+    depends_on "perl" unless OS.mac?
+  
+    resource "fraggenescan" do 
+        url "https://sourceforge.net/projects/fraggenescan/files/FragGeneScan1.31.tar.gz/download"
+        sha256 "cd3212d0f148218eb3b17d24fcd1fc897fb9fee9b2c902682edde29f895f426c"
+    end
+
+    def install
+      ENV.prepend "PERL5LIB", Formula["bioperl"].libexec/"lib/perl5"
+      ENV.prepend_create_path "PERL5LIB", prefix/"perl5/lib/perl5"
+      system "cpanm", "-l", prefix/"perl5", "LWP::Simple"
+  
+      resource("fraggenescan").stage do
+        system "make", "clean"
+        system "make", "fgs"
+      end
+
+      cd "src" do
+          system "make"
+      end
```

Comment:
After running `make` you'll need to install the executables that it built. If there's no `make install` target, then use…
```ruby
bin.install "maxbin2", "any-other-executables-go-here"
```